internal/core/metadata/rest_deviceprofile.go

```diff
@@ -456,7 +509,7 @@ func restGetYamlProfileById(
 	w http.ResponseWriter,
 	r *http.Request,
 	loggingClient logger.LoggingClient,
-	dbClient interfaces.DBClient) {
+	dbClient interfaces.DBClient, errorHandler errorconcept.ErrorHandler) {
```

Comment:
Two parameters on one line needs to be two lines with one parameter each.

Applies elsewhere; not commenting additional instances of this.